recipe/migrations/cuda120.yaml

```diff
@@ -3,7 +3,7 @@ __migrator:
   kind:
     version
   migration_number:
-    1
+    2
```

Comment:
Need to double check this doesn't drop the migrator in migrated feedstocks

Reply:
IIUC [this code path in `conda-smithy`]( https://github.com/conda-forge/conda-smithy/blob/16666edc6c5d6b71fd205075c51edc0f408ab044/conda_smithy/configure_feedstock.py#L2245-L2253 ) continues to use the migrator already in a feedstock when a new `migration_number` occurs. IOW it will wait until the migrator bot add the new migration file (or a user adds it manually)

Removal doesn't occur as long as `migrator_ts` is the same, which is the case here

Reply:
Also IIUC the migrators used by `conda-smithy` [can be overridden by passing `--exclusive-config-file` during `rerender]( https://github.com/conda-forge/conda-smithy/blob/16666edc6c5d6b71fd205075c51edc0f408ab044/conda_smithy/configure_feedstock.py#L2220-L2226 ). So tried this with the [ucx-split-feedstock using latest changes]( https://github.com/conda-forge/ucx-split-feedstock/tree/0cf6eabee7753f9c526b6c679ca59492ce756e24 ) and found no change. So it looks like this is not an issue

Reply:
Also have done a few re-renderings myself and with the bot after this was merged and everything seems to be working correctly